web/admin.py

```
@@ -745,6 +747,41 @@ class QuizOptionAdmin(admin.ModelAdmin):
+            {
+                "fields": (
+                    "title",
+                    "student",
+                    "pdf_type",
+                    "subject",
+                    "assignment",
+                    "due_date",
+                    "description",
+                    "pdf_file",
+                    "file_size_display",
+                )
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)